clients/python/tests/conftest.py

```diff
@@ -1,24 +1,76 @@
+import os
+import time
 from typing import Union
 
 import pytest
+from ml_metadata import errors, metadata_store
 from ml_metadata.proto import (
     ArtifactType,
-    ConnectionConfig,
     ContextType,
     metadata_store_pb2,
 )
+from ml_metadata.proto.metadata_store_pb2 import MetadataStoreClientConfig
 from model_registry.core import ModelRegistryAPIClient
 from model_registry.store.wrapper import MLMDStore
 from model_registry.types import ModelArtifact, ModelVersion, RegisteredModel
+from testcontainers.core.container import DockerContainer
+from testcontainers.core.waiting_utils import wait_for_logs
 
 ProtoTypeType = Union[ArtifactType, ContextType]
 
-
-@pytest.fixture()
-def plain_wrapper() -> MLMDStore:
-    config = ConnectionConfig()
-    config.fake_database.SetInParent()
-    return MLMDStore(config)
+# ruff: noqa: PT021 supported
+@pytest.fixture(scope="session")
+def plain_wrapper(request) -> MLMDStore:
+    model_registry_root_dir = model_registry_root(request)
+    print("Assuming this is the Model Registry root directory:", model_registry_root_dir)
+    shared_volume = model_registry_root_dir / "test/config/ml-metadata"
+    sqlite_db_file = shared_volume / "metadata.sqlite.db"
+    if sqlite_db_file.exists():
+        msg = f"The file {sqlite_db_file} already exists; make sure to cancel it before running these tests."
+        raise FileExistsError(msg)
+    container = DockerContainer("gcr.io/tfx-oss-public/ml_metadata_store_server:1.14.0")
+    container.with_exposed_ports(8080)
+    container.with_volume_mapping(shared_volume, "/tmp/shared", "rw") # noqa this is file target in container
+    container.with_env("METADATA_STORE_SERVER_CONFIG_FILE", "/tmp/shared/conn_config.pb") # noqa this is target in container
+    container.start()
+    wait_for_logs(container, "Server listening on")
+    os.system('docker container ls --format "table {{.ID}}\t{{.Names}}\t{{.Ports}}" -a') # noqa governed test
+    print("waited for logs and port")
+    cfg = MetadataStoreClientConfig()
+    cfg.host = "localhost"
+    cfg.port = int(container.get_exposed_port(8080))
+    print(cfg)
```

Comment:
sorry for commenting this now, you're too quick!

Nit:

```suggestion
    assert container.get_exposed_port(8080)
    cfg = MetadataStoreClientConfig(host="localhost", port=8080)
```

Reply:
Sorry, I do not think this is right.
The exposed port 8080 gets mapped to a bindable port on the system running the Testcontainer.
So your proposed line 40 would not work?
Am I missing something?

Reply:
oh you're right, sorry I misunderstood what you had done here. Resolving...